pages/jobs/[slug].tsx

```diff
@@ -28,8 +34,17 @@ export async function getStaticProps({ params }: { params: { slug: string } }) {
 
 export async function getStaticPaths() {
     const allJobSlugs = await getAllJobSlugs();
-    return {
-        paths: allJobSlugs.map((slug) => `/jobs/${slug}`) ?? [],
-        fallback: 'blocking',
-    };
+    if (allJobSlugs !== null && allJobSlugs !== undefined) {
+        return {
+            paths: allJobSlugs.map((slug) => `/jobs/${slug}`) ?? [],
+            fallback: 'blocking',
+        };
+    } else {
+        // If we can't find any job postings from peopleHR, don't create posting pages
```

Comment:
**Praise**: These comments are really useful especially when being new to nextjs